esx_service/vmdk_ops.py

```diff
@@ -402,8 +402,17 @@ def vol_info(vol_meta, vol_size_info, datastore):
 # Return error, or None for OK
 def removeVMDK(vmdk_path):
     logging.info("*** removeVMDK: %s", vmdk_path)
-    cmd = "{0} {1}".format(VMDK_DELETE_CMD, vmdk_path)
 
+    # Check the current volume status
+    attached, uuid, attach_as = getStatusAttached(vmdk_path)
+    if attached:
+        vm = findVmByUuid(uuid)
+        logging.info("*** removeVMDK: %s is in use by %s VM uuid = %s",
```

Comment:
No, please go ahead and lets put this thru @kerneltime's test and close the issue.